### PR TITLE
Allow middle mouse click to open list row in new window

### PR DESCRIPTION
**Pull request by Samuell1**
This allows to use middle click to open list row in new window. 

If anybody knows how to make it to stop focusing new window, like middlemouse natively does on links. Ideas will be appreciated.

### CODE REVIEW

**Comment by LukeTowers** [comment]
@w20k any ideas on how to prevent the focusing on the new window?

**Comment by w20k** [comment]
@LukeTowers should be something similar. One of the options definitely should work:
```
window.open(href).focus();
//Or
var win = window.open(href);
win.blur();
win.opener.focus();
window.focus();
```

**Comment by bennothommo** [comment]
@w20k @Samuell1 I did a quick bit of research into this - apparently it's frowned upon by browsers because it's a behaviour that is used to sneak in unwanted ads, and other nasty stuff such as bitcoin miners.

If it's something that is really required, we'd have to look into replacing the click functionality of the Lists widget with some form of hyperlinking (if the target is a URL, not JavaScript).

**Comment by Samuell1** [comment]
I think for now this is fine. But if there is way to use anchor links on table columns then it will be cool to have it.

File: modules/system/assets/ui/js/list.rowlink.js [hunk 1]
```diff
@@ -62,8 +62,12 @@
                 }
             }
 
-            $(this).not('.' + options.excludeClass).find('td').not('.' + options.excludeClass).click(function(e) {
+            $(this).not('.' + options.excludeClass).find('td').not('.' + options.excludeClass).click( function (e) {
```
Comment (LukeTowers) [inline modules/system/assets/ui/js/list.rowlink.js]:
```suggestion
            $(this).not('.' + options.excludeClass).find('td').not('.' + options.excludeClass).click(function (e) {
```

**Pull request merged**